net/server.go

```diff
@@ -54,22 +61,23 @@ func newServer(n *net, opts ...grpc.DialOption) (*server, error) {
 	s.opts = append(defaultOpts, opts...)
 
 	if n.conf.PubSub {
-		ps, err := pubsub.NewGossipSub(
+		var err error
+		s.ps, err = pubsub.NewGossipSub(
 			n.ctx,
 			n.host,
-			pubsub.WithMessageSigning(false),
-			pubsub.WithStrictSignatureVerification(false))
```

Comment:
These are deprecated. Also, we really do want message signing here because we're no longer doing it at the proto level.